### PR TITLE
Fixing Windows specific linting error (path shenanigans)

### PR DESCRIPTION
**Pull request by ruohki**
Fixing windows path shenanigans

### CODE REVIEW

**Comment by Clovel** [comment]
Perhaps a more explicit title for this PR would garner more attention from the maintainers.

**Comment by Clovel** [comment]
Can I recommend this approach ?

```js
const remoteCatalogRoot = path.normalize(tropohouse.root);
if(process.platform === 'win32' && remoteCatalogRoot.startsWith('/')) {
  remoteCatalogRoot = remoteCatalogRoot.replace(/^\/(\w)\//, "$1:\\");
}
```

File: types.js [hunk 1]
```diff
@@ -21,7 +21,10 @@ ProjectContext.prototype.getProjectLocalDirectory = function () {
   return oldGetProjectLocalDirectory.apply(this, arguments);
 };
 
-const remoteCatalogRoot = tropohouse.root;
+let remoteCatalogRoot = tropohouse.root;
+if (process.platform == "win32") {
+  remoteCatalogRoot = remoteCatalogRoot.replace(/^\/(\w)\//, "$1:\\").replace(/\//g,'\\');  
+}
```
Comment (Clovel) [inline types.js:27]: Can I recommend this approach ?

```js
const remoteCatalogRoot = path.normalize(tropohouse.root);
if(process.platform === 'win32' && remoteCatalogRoot.startsWith('/')) {
  remoteCatalogRoot = remoteCatalogRoot.replace(/^\/(\w)\//, "$1:\\");
}
```

**Review by Clovel — CHANGES_REQUESTED** [review]

File: types.js (hunk 1 above)
Comment (Clovel) [reply types.js:27]: `path.normalize()`'s behaviour tested like this :

```js
const path = require('path');

const winPath0 = String.raw`C:\Users\Me\repository\meteor\meteor-types`;
const winPath1 = String.raw`C:\Users/Me\\repository\\\meteor\\\\meteor-types`;
const winPath2 = String.raw`C:\\Users/Me\\repository\\\meteor\\\\meteor-types`;

console.log(path.normalize(winPath0)) // C:\Users\Me\repository\meteor\meteor-types
console.log(path.normalize(winPath1)) // C:\Users\Me\repository\meteor\meteor-types
console.log(path.normalize(winPath2)) // C:\Users\Me\repository\meteor\meteor-types
```

Comment (zodern) [reply types.js:27]: Plugins have `Plugin.convertToOsPath` (https://docs.meteor.com/api/packagejs#build-plugin-file-system).  @ruohki if would change it to use this, I'll merge the PR.

Comment (ruohki) [reply types.js:27]: I didn't look too deep into this problem, just made it work for me, as I mainly work on Windows.

**Comment by ruohki** [comment]
@zodern okay, that was a lot easier than expected. :)

**Comment by jbo023** [comment]
Would it be possible to merge this, and publish a new version ?

**Comment by zodern** [comment]
Sorry it took so long. I published version 1.0.12 with the fix.

**Pull request merged**